robot/basestation/static/css/themes/lofi.css

```diff
@@ -0,0 +1,195 @@
+}
+.btn-primary:not(:disabled):not(.disabled).active:focus, .btn-primary:not(:disabled):not(.disabled):active:focus, .show>.btn-primary.dropdown-toggle:focus {
+  box-shadow: 0 0 0 0.2rem rgba(255,255,255,0);
+}
+.btn.focus, .btn:focus {
+  box shadow: white;
+}
+.btn-primary {
+  border-color: white;
+}
+.dropdown-item.active, .dropdown-item:active {
+    color: #fff;
+    text-decoration: none;
+    background-color: #BCD4DD;
+}
+
+/* Bootstrap buttons */
+.btn-primary {
+  color: #fff;
+  background-color: #445b7b;
+  border-color: #445b7b;
+}
+.btn-primary:hover {
+  color: black;
+  background-color: #BCD4DD;
+  border-color: #BCD4DD;
+}
+
+/* Bootstrap forms */
+.form-control:focus {
+  color: white;
+  background-color: #fff;
+  border-color: white;
+  outline: 0;
+  box-shadow: 0 0 0 0.2rem rgba(255,255,255,0);
+}
+
+/* Titles */
+.container-header, thead {
+  color: #E4E1D1;
+  font-weight: bolder;
+}
+.title {
+  color: #E4E1D1;
+}
+
+/*Log buttons (rover.css)*/
+#logs-btn button {
+    background-color: #3D7F7F;
+    border: ridge 2px black;
+}
+
+/* Control Buttons (rover.css) */
+.round-btn {
+  background-color: #3D7F7F;
+  border: ridge 2px black;
+  color: white;
+}
+.rect-btn {
+    background-color: #3D7F7F;
+    border: ridge 2px black;
+    color: white;
+}
+.square-btn {
+    background-color: #3D7F7F;
+    border: ridge 2px black;
+    color: white;
+}
+.round-btn:hover, .rect-btn:hover, #logs-btn button:hover {
+    background-color: #BCD4DD;
+    color: black;
+}
+.control-btn-left {
+    border-right-color: #848282;
+}
+.control-btn-right {
+    border-left-color: #848282;
+}
+.control-btn-mid {
+    background-color: #848282;
+    border-top: ridge 2px #848282;
+    border-bottom: ridge 2px #848282;
+    border-left: ridge 2px #848282;
+    border-right: ridge 2px #848282;
+    color: white;
+}
+#motor-table thead tr:first-child, #motor-table tbody th {
+    background-color: #101148;
+    color: #F0DAFA;
+    border: 2px solid #750000
+}
+#motor-table, #motor-table tr td {
+    border: 2px solid #750000;
+    text-align: center
+}
+.control-main-btn {
+    border-bottom: 0px solid #05072f;
+}
+
+/*Arm controls (arm.css)*/
+.auto-ctrl-grp th, .manual-ctrl-grp th {
+    border-color: #2B343B;
+}
+.auto-ctrl-grp tr:last-child {
+    border-bottom: 0px inset #2B343B;
+    color: #E4E1D1;
+    font-weight: bold;
+    background-color: #2B343B;
+}
+.claw-controls {
+  width:30px;
+  background: none;
+  border:0px
+  solid: #2B343B;
+}
+.claw-controls2 {
+  color: #E4E1D1;
+}
+
+/* Arm Motor Table */
+.table, .table-sm, .table-bordered {
+  background: #EFEFEF;
+}
+#motor-table thead tr:first-child, #motor-table tbody th {
+  background-color: #C34641;
+  color: white;
+  border: 2px solid black;
+}
+#motor-table, #motor-table tr td {
+  border: 2px solid black;
+}
+.table-text {
+  color: black;
+}
+
+/* Navigation Text */
+.navigation {
+  color: #E4E1D1;
+}
+.row {
+  color: #E4E1D1;
+}
+
+/* PDS Text */
```

Comment:
I like how you're grouping everything by component/page. Since the name `pds-text` makes it pretty obvious that this is for the PDS text perhaps the comment here can be changed to a more general `/* PDS */` instead? Same idea for `/* Navigation Text */` above, though in that case it is not obvious that the selectors are meant just for text.

Reply:
I agree. Fixed